libfreerdp/core/freerdp.c

```diff
@@ -498,7 +498,7 @@ BOOL freerdp_context_new(freerdp* instance)
 
 	IFCALLRET(instance->ContextNew, ret, instance, instance->context);
 
-	if (ret)
+	if (!ret)
```

Comment:
This is definitely wrong. If ret is FALSE here this means that instance->ContextNew is set and failed.
This change would break all clients!